app/views/recipes/new.html.erb

```diff
@@ -1,9 +1,9 @@
-<div class="mx-auto md:w-2/3 w-full">
-  <h1 class="font-bold text-4xl">New Recipe</h1>
+<div class="mx-auto md:w-4/5 w-full form-container lg:p-40 p-5">
+  <h1 class="font-bold text-2xl lg:text-4xl text-teal-900 text-center text-start">Create your recipe</h1>
```

Comment:
I didn't touch those since we were primarily focused on the form. I can add the same padding to the show and edit on this ticket as well if you want. Not a big deal. I found the container, really helped the form align better. I'll test to see if it works well on the show page, the width might need to be different depending on the way the layouts shift.